README.md

```diff
@@ -252,22 +238,9 @@ Those resources are not destroyed automatically as part of undeploying the stack
 
 ### 2. Undeploy Configurations in the Project
 
-Undeploy each configuration in the project, one by one, via UI, starting from the "6 - Sample RAG app configuration" and working your way up in the stack up to, and inclusive of "2a - Security Service - Key Management". Wait for full undeployment of a configuration before starting to undeploy the next configuration up in the stack.
-
-### 3. Delete Reclamation Claims
```

Comment:
Does "Stack undeploy" take care of the resource reclamation? If not, this will still be an issue - e.g. if you used trial Secrets Manager plan, undeployed it, then you will not be able to deploy another trial instance until reclamation is deleted. Same with destroying a resource group where reclaimed resources are.

Reply:
@in-1911 - no, stack undeploy does not take care of resource reclamation. There is a change in BSS where it is now possible to delete a resource group that has got reclamations pending. The secret manager aspect is already described in a separate section https://github.com/terraform-ibm-modules/stack-retrieval-augmented-generation?tab=readme-ov-file#limitations-with-the-trial-secret-manager-offering